apps/wing/src/commands/test/test.ts

```diff
@@ -134,7 +140,11 @@ export async function test(entrypoints: string[], options: TestOptions): Promise
       });
     }
   };
-  await Promise.all(selectedEntrypoints.map(testFile));
+
+  await PromisePool.withConcurrency(options.parallel || selectedEntrypoints.length)
```

Comment:
nit

```suggestion
  await PromisePool.withConcurrency(options.parallel ?? selectedEntrypoints.length)
```

Reply:
actually not- if options.parallel is 0 we want no limit...